app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
```

Comment:
> But I think it will even increase your merge chance if you
> 
> * refactor middleware to use the mixin (this should be mostly a code movement PR)
> * use the mixin here

@durandom I discussed this in the past with @pilhuhn and @lucasponce and anything related to the middleware provider picking up these changes will be handled by the middleware team (of course we'll provider assistance).